next.config.mjs

```diff
@@ -7,7 +7,7 @@ import "./env.mjs"
 const nextConfig = {
   reactStrictMode: true,
   images: {
-    domains: ["avatars.githubusercontent.com", "f005.backblazeb2.com", "embloy2.blob.core.windows.net", "embloy.s3.eu-central-1.amazonaws.com"],
+    domains: ["avatars.githubusercontent.com", "f005.backblazeb2.com", "embloy2.blob.core.windows.net", "embloy.s3.eu-central-1.amazonaws.com", "api.embloy.com"],
```

Comment:
Do we need this?
All images should be stored on embloy.s3

Reply:
ich nutze nur das was in dem json von der api steht lol

Reply:
Ja aber eigentlich served api.embloy.com keine dateien, oder haben wir etwa user mit image attachments mit api.embloy.com als host?

Reply:
doch. hat mich ultra gewndert

Reply:
dann schick mir mal bitte die user ids